interface/backend/cases/models.py

```diff
@@ -10,6 +10,48 @@
 from rest_framework import serializers
 
 
+def django_enum(cls):
+    # decorator needed to enable enums in django templates
+    cls.do_not_call_in_templates = True
+    return cls
+
+
+class PleuralSpace(models.Model):
+    created = models.DateTimeField(default=timezone.now)
+
+    @unique  # ensures all variables are unique
+    @django_enum
+    class TypesOne(IntEnum):
```

Comment:
From experience I have a very strong preference for putting enumerations into a separate ``enums.py`` file alongside the ``models.py`` — could you try this approach and see how it looks/works for you? It might not be suitable, but definitely worth having a look at...

Reply:
Done: [`enum.py`](https://github.com/concept-to-clinic/concept-to-clinic/pull/185/commits/d4e5a767b0dc943a624d9ee577a5a1dbe990ac4d#diff-11ad5b67ec28e05935f1d6da87a683e8), looks much better now, thanks!